ui/src/app/edge/live/common/grid/modal/modal.ts

```diff
@@ -10,36 +10,37 @@ import { Role } from 'src/app/shared/type/role';
 })
 export class ModalComponent extends AbstractFormlyComponent {
 
-  protected generateView(edgeId: string, config: EdgeConfig, role: Role): { title: string; lines: OeFormlyField[]; } {
-    const meters = config.getComponentsImplementingNature("io.openems.edge.meter.api.ElectricityMeter")
-      .filter(comp => comp.isEnabled && config.isTypeGrid(comp));
+  public override generateView(edgeId: string, config: EdgeConfig, role: Role, translate: TranslateService): OeFormlyView {
```

Comment:
`protected override` müsste reichen, oder?

Apropos override: ich würde gerne die Option [noImplicitOverride](https://www.typescriptlang.org/tsconfig#noImplicitOverride) aktivieren, damit `override` verpflichtend wird: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html#override-and-the---noimplicitoverride-flag

Reply:
wird in den nächsten PR mitreingenommen